ignite/distributed/comp_models/xla.py

```diff
@@ -159,5 +160,11 @@ def _do_broadcast(self, tensor: torch.Tensor, src: int) -> torch.Tensor:
             xm.all_reduce("sum", [tensor])
             return tensor
 
-        def barrier(self) -> None:
-            xm.rendezvous("barrier")
+        def barrier(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Let's make "tag" explicit:
```suggestion
        def barrier(self, *args: Any, tag="barrier", **kwargs: Any) -> None:
```

Reply:
I've Tried to make this fix, will work then on tests ...